app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -57,12 +58,15 @@ case class GlobalAttributeForAPI(val globalAttributeId: Int,
         "agree_count" -> agreeCount,
         "disagree_count" -> disagreeCount,
         "notsure_count" -> notsureCount
+      ),
+      "users" -> Json.obj(
```

Comment:
Let's just add a `user_ids` field to the `properties` object instead of making something that is separate from `properties`.